tests/python/driver/tvmc/test_frontends.py

```diff
@@ -211,3 +212,43 @@ def test_load_model___wrong_language__to_pytorch(tflite_mobilenet_v1_1_quant):
             model_format="pytorch",
             shape_dict={"input": [1, 3, 224, 224]},
         )
+
+
+@mock.patch("tvm.driver.tvmc.frontends.import_keras")
+def test_import_keras_friendly_message(import_keras_mock, keras_resnet50):
+    import_keras_mock.side_effect = TVMCException("keras is not installed")
+
+    with pytest.raises(TVMCException) as e:
+        _ = tvmc.frontends.load_model(keras_resnet50, model_format="keras")
+
+
+@mock.patch("tvm.driver.tvmc.frontends.import_onnx")
+def test_import_onnx_friendly_message(import_onnx_mock, onnx_resnet50):
```

Comment:
```suggestion
def test_import_onnx_friendly_message(import_onnx_mock, onnx_resnet50):
    # some CI environments wont offer onnx, so skip in case it is not present
    pytest.importorskip("onnx")
```